core/base/mergeTreeClustering/PathMappingDistance.h

```diff
@@ -0,0 +1,360 @@
+/// \ingroup base
+/// \class PathMappingDistance
+/// \author Florian Wetzels (wetzels@cs.uni-kl.de)
+/// \date 2022.
+///
+/// This module defines the %PathMappingDistance class that computes distances
+/// between two merge trees.
+///
+/// \b Related \b publication \n
+/// "A Deformation-based Edit Distance for Merge Trees" \n
+/// Florian Wetzels, Christoph Garth.\n
+/// TBP.
+
+#pragma once
+
+#include <set>
+#include <vector>
+
+#include <algorithm>
+#include <cfloat>
+#include <chrono>
+#include <cmath>
+#include <iostream>
+#include <limits>
+#include <set>
+#include <stack>
+#include <tuple>
+#include <vector>
+
+// ttk common includes
+#include <AssignmentAuction.h>
+#include <AssignmentExhaustive.h>
+#include <AssignmentMunkres.h>
+#include <Debug.h>
+
+namespace ttk {
+
+  class PathMappingDistance : virtual public Debug {
+
+  private:
+    int baseMetric = 0;
```

Comment:
usually, in ttk, the member variables of a class, within the core/base/ subdirectories, end with a final _ (in order to distinguish them from local variables, later in the functions).
see assignmentSolverID_ for instance.
that'd be great if you could follow that convention.